### ISSUE TITLE
Author Biography Template Should be Marked up as an HCard

### ISSUE THREAD

**Issue by dshanske** [WordPress/twentysixteen#73]
http://microformats.org/wiki/hcard 

HCards are recognized by a variety of sites and the formatting for them appear in various parts of WordPress.

**Comment by karmatosed** [comment]
I'm not sure this has to have vcard. What do you think @iamtakashi ?

**Comment by iamtakashi** [comment]
I don't have a strong opinion on this.

@dshanske, since you opened the issue, any feed back on the PR?

**Comment by dshanske** [comment]
What is the argument against it? We are representing an author, so it does apply.

**Comment by iamtakashi** [comment]
I did not say I'm against it. 

I just asked your feedback on the PR #88 because you seems to have strong opinions on microformats and It's always helpful for us to hear "+1" or "Looks good" from the person who opened an issue.

**Comment by dshanske** [comment]
I think I didn't write what I meant clearly(that is what you get when you jot a quick note on your phone). I was trying to figure out if there was a concern that it wasn't an hcard, and thus shouldn't be marked up with vcard.

So, I ran the entire thing through a parser...with and without the change. And I found a conflict.

The vcard in the entry metadata and the vcard in the author bio make duplicative vcards on a single page. Therefore, I would say this  one can't be closed. 

Although looking at the result, the bio and the meta are both showing the name and avatar, so I'm not sure which one should win out there. 

This might require more thought.
